coredns/datadog_checks/coredns/coredns.py

```diff
@@ -0,0 +1,110 @@
+# (C) Datadog, Inc. 2018
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+
+from datadog_checks.checks.openmetrics import OpenMetricsBaseCheck
+from datadog_checks.errors import CheckException
+
+
+class CoreDNSCheck(OpenMetricsBaseCheck):
+    """
+    Collect CoreDNS metrics from its Prometheus endpoint
+    """
+
+    # Set up metric_transformers
+    METRIC_TRANSFORMERS = {}
+
+    def __init__(self, name, init_config, agentConfig, instances=None):
+        # Set up metric_transformers
+        self.METRIC_TRANSFORMERS = {
+            # Primarily, metrics are emitted by the prometheus plugin: https://coredns.io/plugins/metrics/
+            # Note: the count metrics were moved to specific functions
+            # below to be submitted as both gauges and monotonic_counts
+            # 'coredns_dns_response_rcode_count_total': self.coredns_dns_response_rcode_count_total,
```

Comment:
Since we're not using the transformer functions, we can just get rid of this dictionary.